localization/english/wc_buildings_l_english.yml

```
@@ -917,9 +917,24 @@
  building_type_shaladrassil_desc:0 "A World Tree under which the surviving satyrs from the War of the Ancients were placed in an eternal sleep deep beneath the roots of."
  building_shaladrassil:0 "Shaladrassil"
  building_shaladrassil_desc:0 "A World Tree under which the surviving satyrs from the War of the Ancients were placed in an eternal sleep deep beneath the roots of."
- 
+ building_type_black_rook_hold_01:0 "Black Rook Hold"
+ building_black_rook_hold_01:0 "$building_type_black_rook_hold_01$"
+ building_black_rook_hold_01_desc:0 "Held by Lord Kur'talos Ravencrest, Black Rook Hold served as a unassailable fortress against the Burning Legion in the War of the Ancients. Being forged by ancient elven craftsmen from Val'sharah's towering peak, this nearly invincible fortress is a sight to behold."
+ building_black_rook_hold_02:0 "$building_black_rook_hold_01$"
+ building_black_rook_hold_02_desc:0 "$building_black_rook_hold_01_desc$"
+
+ building_type_neltharions_lair_01:0 "Neltharion's Lair"
+ building_neltharions_lair_01:0 "$building_type_neltharions_lair_01$"
+ building_neltharions_lair_01_desc:0 "Neltharion's Lair houses the Pillar of Creation, specifically the Hammer of Khaz'goroth. This cavern was formerly the abode of the Earth Warder Neltharion. Following his descent, the drogbar, who once revered this mighty dragon, established their capital in this ancient lair. Presently Dargrul, the chieftain of the drogbar, is amassing a formidable army in the depths. Empowered by the Hammer of Khaz'goroth, this force poses a significant risk; if not confronted, it could surge from the mountainside, wreaking havoc on Highmountain and jeopardizing the overall security of Azeroth."
```

```suggestion
 building_neltharions_lair_01_desc:0 "Neltharion's Lair houses the Pillar of Creation, specifically the Hammer of Khaz'goroth. This cavern was formerly the abode of the Earth Warder Neltharion. Following his descent, the drogbar, who once revered this mighty dragon, established their capital in this ancient lair. Guarded by massive worms and earth elementals, this underground fortress could provide the space, safety, and resources to equip and train an army of thousands."
```
i would avoid references to WoW events in the present tense because they are unlikely to actually happen in the provided timestamps.